interfaces/apparmor/backend_test.go

```diff
@@ -688,6 +688,7 @@ const commonPrefix = `
 @{SNAP_NAME}="samba"
 # This is a snap name with instance key
 @{SNAP_INSTANCE_NAME}="samba"
+@{SNAP_COMMAND_NAME}="smbd"
```

Comment:
What is the value for hooks?

I like the idea but I wonder if instead we should simply have $SNAP_SECURITY_TAG as a generalization of the concept. I don't mind landing this if we intend to iterate as this is an entirely internal variable.

Reply:
I implemented it differently than `@{SNAP_SECURITY_TAG}` because unless you are in a child profile, `@{profile_name}` will be the same as `@{SNAP_SECURITY_TAG}`. Because `@{profile_name}` should always be used instead of `@{SNAP_SECURITY_TAG}` outside of a child profile and is almost always the correct choice when in a child profile, the subtlety of when to use `@{SNAP_SECURITY_TAG}` correctly could lead to bugs or hinder readability.

Instead, I continue to use `@{SNAP_COMMAND_NAME}` but prepend `hook.` in addContent(). I think `@{SNAP_COMMAND_NAME}` still works as a variable name, but let me know if there is something better and I can change it.